examples/src/main/python/streaming/hdfs_wordcount.py

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-"""
+r"""
```

Comment:
Seems this one doesn't have a balckslash

Reply:
Yes.  Thanks for pointing it out.  Fixed.